### DISCUSSION TITLE
/v2/zh/guide/layout/navbar.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#193] [Announcements]
# /v2/zh/guide/layout/navbar.html

一个具有强大功能的 vuepress 主题✨

https://vuepress-theme-hope.github.io/v2/zh/guide/layout/navbar.html

**Comment by wlsh123** [comment]
请问导航栏的icon是相对路径吗。我怎么导都不对呢
